### ISSUE TITLE
PromethION data de novo assembly?

### ISSUE THREAD

**Issue by Samvkes** [genome-in-a-bottle/giab_data_indexes#23]
Hi, in the README for the ONT-PromethION datasets (https://ftp-trace.ncbi.nlm.nih.gov/ReferenceSamples/giab/data/AshkenazimTrio/HG002_NA24385_son/UCSC_Ultralong_OxfordNanopore_Promethion/) under 'Data Processing Methods', alignment of the called reads is mentioned. But in the linked paper (https://doi.org/10.1101/715722) it's explained that the dataset was assembled de novo, only doing alignment afterward for benchmarking (unless I'm misunderstanding). 

Also in the README under the 'Data Processing Methods'-header,  a newer version of Guppy is mentioned than the one used in the paper, which suggests to me that that part was added more recently, and it contains no information on assembly at all. Does that mean that newer versions of the data are no longer generated de novo?

**Comment by jzook** [comment]
Yes, these are only mapped reads and not assemblies. ONT assembly methods
are quickly evolving, but one recent preprint with an assembly of HG002 ONT
reads is at https://doi.org/10.1101/2023.01.12.523790
